src/js/MazeEntity.js

```diff
@@ -2,6 +2,18 @@ class MazeEntity {
     constructor(maze, row, col) {
         Object.assign(this, {maze, row, col});
     }
+    getNeighboringEntitiesDict(distance){
```

Comment:
maybe rename `getNeighborsAtDist() `(or revert to this name, since it sounds like you had previously called it that ;) )